### PR TITLE
Update AZRTOS instructions

### PR DESCRIPTION
**Pull request by hihigupt**
Update AZRTOS instructions

### CODE REVIEW

**Comment by hihigupt** [comment]
@ericmitt is added to the review. #Closed

**Comment by hihigupt** [comment]
@liydu is added to the review. #Closed

**Comment by hihigupt** [comment]
@yuxin-azrtos is added to the review. #Closed

**Comment by hihigupt** [comment]
@TiejunMS is added to the review. #Closed

**Comment by hihigupt** [comment]
@bo-ms is added to the review. #Closed

File: bugbash.md [hunk 1]
```diff
@@ -79,7 +79,15 @@ All quickstarts and tutorials are in the [IoT Plug and Play documentation](https
 
 #### EmbeddedC and Azure RTOS
 
-You can also test the **Quickstart on Azure RTOS for STM32L475 device**, [download this zip file](https://github.com/ericmitt/IotPnp/AzureRTOS_GA_Bugbash.zip) and follow the instructions in the readme pdf file.
+You can also test the **Quickstart on Azure RTOS for hardware we support**, [download this zip file based on the hardware and IDE](https://microsoft-my.sharepoint.com/:f:/p/liydu/EjI0F-b7aLdNhkT6AGnlvh4B9xixZzjN77CVO1zWZSNp4Q?e=F6pbWn) and follow the instructions in the readme pdf file.
```
Comment (TiejunZhou-CN) [inline bugbash.md]: Could you mention where to find the readme pdf file? In `docs` folder.

**Review by TiejunZhou-CN — APPROVED** [review]

File: bugbash.md (hunk 1 above)
Comment (hihigupt) [reply bugbash.md]: updated

File: bugbash.md [hunk 2]
```diff
@@ -79,7 +79,15 @@ All quickstarts and tutorials are in the [IoT Plug and Play documentation](https
 
 #### EmbeddedC and Azure RTOS
```
Comment (bochen-1) [inline bugbash.md]: How about "Embedded C SDK and Azure RTOS"

**Review by bochen-1 — APPROVED** [review]

File: bugbash.md (hunk 2 above)
Comment (hihigupt) [reply bugbash.md]: sure

**Review by ericmitt — APPROVED** [review]

**Pull request merged**